packages/cli-kit/src/public/node/base-command.ts

```diff
@@ -8,7 +8,8 @@ import {JsonMap} from '../../private/common/json.js'
 import {outputContent, outputInfo, outputToken} from '../../public/node/output.js'
 import {hashString} from '../../public/node/crypto.js'
 import {isTruthy} from '../../private/node/context/utilities.js'
-import {Command, Interfaces} from '@oclif/core'
+import {Command} from '@oclif/core'
+import {FlagOutput, Input, ParserOutput, FlagInput, ArgOutput} from '@oclif/core/lib/interfaces/parser.js'
```

Comment:
can we add a todo here to note that we should try to fix this?

Reply:
I'll create an issue and work on it now, it will imply breaking changes from cli-kit so it has priority